### ISSUE TITLE
second layer of oserver

### ISSUE THREAD

**Issue by weiyuan-jiang** [GEOS-ESM/MAPL#467]
So far it is fast for the App to dump the data to the oserver and move on to the other tasks. However, the oserver will not respond to the App until it finishes writing the data to the disk.  That creates delay when the App dumps small data after big data. It has to wait for the big data to be written. To solve this issue, we create two sets of oservers, the bigger one ( with more processes) will take the big data and the small one take the small data. It works. However it is hard ( or may be impossible) for users to configure such oservers.  Also it is inflexible since it has to be decided beforehand.  We change the strategy by launching the second layer of the writers.  Th oserver will forward data to an idle writer instead of writing the data directly to the disk. The forwarding is through networking, which is supposed to be faster that writing.

**Comment by mathomp4** [comment]
@weiyuan-jiang Can this be closed? Maybe close this and open a new issue?

**Comment by mathomp4** [comment]
Reexamined on 2021-Apr-12 with @bena-nasa and @tclune
